### ISSUE TITLE
Injection problem

### ISSUE THREAD

**Issue by Nikutsuke** [danielkrupinski/Osiris#962]
Hey guys 2 days ago i upgraded my windows 8.1 to windows 10 and did clean installation and now i cant inject this cheat into csgo i use lit injector and when i press inject it says its injected but i dont get notification in csgo and when i press insert nothing happens wanna tell me what im missing or idk?

**Comment by rosesware** [comment]
Try another injector.

**Comment by VuBrian22** [comment]
Try SazZ's Injector as the Lit Injector is no longer being updated and maintained.

**Comment by thomas0aotian** [comment]
Danial's memject the best. Sry

**Comment by VuBrian22** [comment]
> Danial's memject the best. Sry

Yeah, however he's currently on cheater.fun and the SazZ's injector is more convenient for him. People are lazy, lets face it. He'd rather have an on-par undetected injector with one click of a button, Not with many clicks.

**Comment by xAkiraMiura** [comment]
> 
> 
> Danial's memject the best. Sry

Considering you have to manually hex edit and hard code your DLL on every update into the source and then recompile it, I think it is easily the worst injector as is unless you know how to adapt it to what he made it for (not hard coding your paste).

You could use daniel's other injector that doesn't do this hard coded hex trash.
Making a simple injector isn't hard, many tutorials exist.

**Comment by thomas0aotian** [comment]
@xAkiraMiura yeah lazy people are always shit in their life thats why u have 0 accomplishment in ur entire life because of the laziness. Memject isn't that hard, danial had every thing u need in his repo.

**Comment by xAkiraMiura** [comment]
why you are wrong:
[Stackoverflow](https://stackoverflow.com/a/828220)
[Stackexchange](https://gamedev.stackexchange.com/questions/84221/why-is-it-bad-to-hard-code-content)
[Google search](https://www.google.com/search?q=why+is+it+bad+to+hard-code+in+programming)

@danielkrupinski Could you take temperary action against @thomas0aotion? On top of what I have said before they don't contribute anything to threads and actively post misinformation.
They are trying to start a flame war rather than be mature and contribute. Additionally they have tried to get stolen/pasted code into the project.
sample 1: https://github.com/danielkrupinski/Osiris/issues/941#issuecomment-572427928
sample 2: https://github.com/danielkrupinski/Osiris/issues/962#issuecomment-572633063

_imo they are trying to make anti-patterns to make the project so unbearable to work on so it dies but cant assume malice over incompetence_

**Comment by thomas0aotian** [comment]
@xAkiraMiura seems like only thing u do is to piss everyone here off and be like a complete retard. Every single one of ur retarded comments are pretty offensive. Why don't u just get back to ur mom and stop crying on the web? Somewhat 13-year-old child

**Comment by thomas0aotian** [comment]
Boomer Ignored
